File: ROID-Forum-Server/ROID-Forum-Server/Convert Server Code/GMGServer/bin/Server/GenMsg.dart
library Server.GenMsg;
import 'dart:convert';

class GenMsg {
  static String Login(bool success) {
    Map m = new Map();
    m["Title"] = "Login";
    m["Status"] = success;
    return JSON.encode(m);
  }
  static String Register(bool success) {
    Map m = new Map();
    m["Title"] = "Register";
    m["Status"] = success;
    return JSON.encode(m);
  }
  static String Name(String status) {
    Map m = new Map();
    m["Title"] = "Name";
    m["Status"] = status;
    return JSON.encode(m);
  }
  static String Enter(String game) {
    Map m = new Map();
    m["Title"] = "Enter";
    m["Game"] = game;
    return JSON.encode(m);
  }
}

File: ROID-Forum-Server/ROID-Forum-Server/GenMsg.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
namespace ROIDForumServer
{
    public class GenMsg
    {
        public static String Login(bool success)
        {
            Dictionary<string, object> m = new Dictionary<string, object>();
            m["Title"] = "Login";
            m["Status"] = success;
            return JsonConvert.SerializeObject(m);
        }
        public static String Register(bool success)
        {
            Dictionary<string, object> m = new Dictionary<string, object>();
            m["Title"] = "Register";
            m["Status"] = success;
            return JsonConvert.SerializeObject(m);
        }
        public static String Name(String status)
        {
            Dictionary<string, object> m = new Dictionary<string, object>();
            m["Title"] = "Name";
            m["Status"] = status;
            return JsonConvert.SerializeObject(m);
        }
        public static String Enter(String game)
        {
            Dictionary<string, object> m = new Dictionary<string, object>();
            m["Title"] = "Enter";
            m["Game"] = game;
            return JsonConvert.SerializeObject(m);
        }
    }
}
